.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -129,9 +145,11 @@ jobs:
       contents: 'read'
       id-token: 'write'
     steps:
-      - name: Echo Rawls version
+      - name: Echo Rawls version and version template
         run: |
-          echo '${{ needs.rawls-build-tag-publish-job.outputs.custom-version-json }}'
+          echo "built custom Rawls=${{ inputs.build-branch}}"
+          echo "custom build Rawls version=${{ needs.rawls-build-tag-publish-job.outputs.custom-version-json }}"
+          echo "version-template=${{ inputs.bee-version-template }}"
```

Comment:
Example when we build an image from dev: https://github.com/broadinstitute/rawls/actions/runs/7213989936/job/19655405356

Example from when we don't build a custom image: https://github.com/broadinstitute/rawls/actions/runs/7214169971/job/19655675374